.env

```diff
@@ -1 +1,3 @@
-APP_PORT=4000
\ No newline at end of file
+APP_PORT=4000
```

Comment:
Add this env file in the .gitignore and remove it from your repo

Reply:
Done as requested.